src/styles/theme/default.theme.ts

```diff
@@ -24,6 +24,8 @@ const theme: OakTheme = {
     oakGrey4: "#575756",
     oakGrey5: "#2D2D2D",
     oakGrey6: "#1B1B1B",
+    oakGrey30: "#E4E4E4",
```

Comment:
should these new colours be in the //oak new colours section below?

Reply:
I'm not sure, @Monika-Koziol are these new design kit colors ?